local_groups/migrations/0061_auto_20180207_0131.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.2 on 2018-02-07 01:31
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('local_groups', '0060_auto_20171213_2208'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
```

Comment:
are you sure this is safe? going to wipe out existing data?

Reply:
We only have four groups marked, and we're manually remarking them anyways, so wanted to avoid having to do separate deploys due to time-sensitivity.

Reply:
I suggest let's remove it from app logic in one deploy, and drop from database in later deploy, to avoid downtime. A/B deploy issue and preserving legacy data issue are two separate things. If we don't need the legacy data that's fine, but we still should avoid throwing errors during deploy.